docs/containerd-env-setup.md

```diff
@@ -74,21 +74,23 @@ sudo rm -rf /var/lib/containerd-nydus
 ```
 
 3. Start `containerd-nydus-grpc` (nydus snapshotter):
+Optionally, a TOML nydus-snapshotter configuration file can be provided by appending `--config <CONFIG>` when starting nydus-snapshotter if you want fine-grained control items. An example configuration file can be found [here](https://github.com/containerd/nydus-snapshotter/blob/main/misc/snapshotter/config.toml)
 
 ```bash
 sudo /usr/bin/containerd-nydus-grpc \
-    --config-path /etc/nydus/nydusd-config.fusedev.json \
+    --fs-driver fusedev \
+    --nydusd-config /etc/nydus/nydusd-config.fusedev.json \
     --log-to-stdout
 ```
 
 ## Configure and Start Containerd
 
-Nydus uses two features of containerd:
+Nydus depends on two features of Containerd:
 
-- remote snapshotter
-- snapshotter annotations
+- Remote snapshotter plugin support
+- Pass the annotations of image layer to the remote snapshotter
```

Comment:
- Support remote snapshotter plugin
- Support passing image-related annotations to remote snapshotter

Reply:
fixed